### ISSUE TITLE
No longer present in menu since VS 2017 since 15.6.4 update

### ISSUE THREAD

**Issue by MrKennie** [madskristensen/ImageOptimizer#39]
VS 2017 15.6.4
Installed via Web Essentials 2017 package

The option to optimize images is no longer present in the menu when right clicking an image. I didn't really pay attention as to exactly when it vanished but the only thing that has changed recently is the 15.6.4 update. This is with a Core 2.0 MVC project open.

**Comment by madskristensen** [comment]
Does it show up correctly for you on other project types besides ASP.NET Core 2.0 projects? It shows up for me correctly

**Comment by MrKennie** [comment]
Not showing in any of my older MVC projects either. I've also reinstalled web essentials but still no show.

**Comment by MrKennie** [comment]
I think I've found the problem. I removed web essentials but found that various packages were still installed including Image Optimizer except some were marked as disabled. I think something has gone wrong with my installation as the web essentials installer is no longer behaving the way it normally does where it lists all packages it's about to install. I guess this is no longer an IO issue.

I can confirm however that IO is present and correct when installed on its own.
